src/main/java/salesforce/ui/pages/lightning/worktype/NewWorkTypePage.java

```diff
@@ -40,6 +46,7 @@ public void setInputField(final String fieldName, final String fieldValue) {
             String xpath = String.format("//*[contains(text(),'%s')]/../..//*[@type='text']", fieldName);
             webElementAction.setInputField(By.xpath(xpath), fieldValue);
         }
+        webElementAction.clickByXpath("//div[@class='actionsContainer']");
```

Comment:
Could this locator be set as an attribute?

Reply:
I will refactor in the next pr